src/06-errors_and_files/01-either.md

```diff
@@ -85,8 +85,8 @@ max3chars x y z =
 
 
 The `Functor` and `Applicative` interfaces of `Either a` allow us to
-apply functions on the payload values of `Either a` types (where the `a` is the
-same between all of applied values them) and **delay** the error handling to a
+apply functions to the payload values of `Either a` types (where the `a` is the
+same type for all of the payload values) and **delay** the error handling to a
```

Comment:
```suggestion
apply functions to the payload values and **delay** the error handling to a
```

Reply:
I liked that you were pointing out that the type variable in the "Either a"s had to be the same. Maybe add a parenthesis like this: "apply functions to the payload values (all are the same type)"

Reply:
I was thinking that this can be skipped here because it is explained in depth later in the chapter. I feel it is a bit awkward to explain here. If you think this is worth mentioning anyway I can spend some time thinking about how to explain this better after we merge the PR.

Reply:
I agree that it can be skipped here.